modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of the AWS Load Balancer Controller using ALB Operator
+
+After installing the Operator, you can create an instance of the `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The Operator reconciles only the resource with the name `cluster`.
+
+.Prerequisites
+
+. You have created the `echoserver` namespace.
+
+.Procedure
+
+. Create a `AWSLoadBalancerController` resource:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1alpha1
+kind: AWSLoadBalancerController <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: Auto <3>
+  additionalResourceTags: <4>
+    example.org/cost-center: 5113232
+    example.org/security-scope: staging
+  ingressClass: cloud <5>
+  config:
+    replicas: 2 <6>
+  enabledAddons: <7>
+    - AWSWAFv2 <8>
+----
+
+<1> Defines the `AWSLoadBalancerControlller` resource.
+<2> Defines the `aws-load-balancer-controller` instance name. This instance name gets added as a suffix to all related resources.
+<3> Valid options are `Auto` and `Manual`. When the value is set to `Auto` the Operator attempts to determine the subnets that belong to the cluster and tags them appropriately. The Operator cannot determine the role correctly if the internal subnet tags are not present on internal subnet. If you installed your cluster on user-provided Infrastructure, you can manually tag the subnets with the appropriate role tags and set the subnet tagging policy to `Manual`.   
+<4> Defines the tags used by the controller when it provisions AWS resources.
+<5> The default value for this field is `alb`. The Operator provisions an `IngressClass` with the same name if it does not exist.
+<6> Specifies the number of replicas of the controller.
+<7> Specifies add-ons for AWS load balancers, which get specified through annotations.
+<8> Enables the `alb.ingress.kubernetes.io/wafv2-acl-arn` annotation.
+
+. After the AWS Load Balancer Controller is running, create a `deployment`:
```

Comment:
The numbering of this step seems to be a bit off in the preview. I presume it should be step 2 (Step 1 being [Create a AWSLoadBalancerController resource:](https://github.com/openshift/openshift-docs/pull/45430/files#diff-6941152beafbe396f2745a1242192c6d32a2141113be19fe7c85e9a4acee698aR17) )

Reply:
Actually, the preview is a bit older, I will add the updated local preview asap.